packages/gatsby-source-prismic/src/plugin-options-schema.ts

```diff
@@ -71,7 +139,7 @@ const externalValidationProgram = (
       pipe(
         scope.repository.types,
         R.keys,
-        A.difference(s.Eq)(scope.schemaTypes),
+        A.difference(string.Eq)(scope.schemaTypes),
```

Comment:
string doesn’t confuse the compiler ... nice

Reply:
Haha it felt weird writing `string` as a variable. This may end up being changed to `str` just to reduce confusion.